com.hack_computer.Clippy.Extension.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id": "com.hack_computer.Clippy.Extension",
+    "branch": "stable",
```

Comment:
please remove this entirely. it's set by flathub automatically. Also, having this will break your build in the beta environment.